dataikuapi/dss/plugin.py

```diff
@@ -147,4 +175,148 @@ def put_file(self, path, f):
         data = f.read() # eat it all, because making it work with a path variable and a MultifilePart in swing looks complicated
         return self.client._perform_empty("POST", "/plugins/%s/contents/%s" % (self.plugin_id, path), raw_body=data)
 
-        
\ No newline at end of file
+
+class DSSPluginUsage(object):
+    """
+    Information on a usage of an element of a plugin.
+    """
+    def __init__(self, json_object):
+        self._json_object = json_object
```

Comment:
[Nitpick] I would rename the field as `_dict` or `_data` as it is not JSON anymore. Same for argument.

Reply:
ok.